client.lua

```diff
@@ -373,7 +373,7 @@ end)
 
 --- Starts the test drive. If vehicle parameter is not provided then the test drive will start with the closest vehicle to the player.
 --- @param vehicle number | nil
-RegisterNetEvent('qb-vehicleshop:client:testDrive', function(vehicle)
+RegisterNetEvent('qb-vehicleshop:client:TestDrive', function(vehicle)
```

Comment:
camelCase is correct. And there are two callers of this event that call it with two different casings. Change the PascalCase caller to camelCase instead of changing the event name.

Reply:
Based on discord conversation NetEvents considered global, so PascalCase Remained. Server side event typo is fixed.